consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java

```diff
@@ -81,20 +77,13 @@ public RoundChangeCertificate createRoundChangeCertificate() {
   @VisibleForTesting
   final Map<ConsensusRoundIdentifier, RoundChangeStatus> roundChangeCache = Maps.newHashMap();
 
-  private final int quorumSize;
+  private final long quorum;
```

Comment:
Bit odd we are using quorum here and quorumSize elsewhere. Though it probably should be just quorum everywhere.

Reply:
yeah agreed - will do a quick round up.